src/components/LoanCards/LoanCardImage.vue

```diff
@@ -0,0 +1,63 @@
+<!--
+	issues:
+
+	* tracking click on image - do here or bubble up?
+		kvtrackevent="Lending|click-Read more|Photo|{{id}}|true"
```

Comment:
Doing here makes sense. We have a global directive for supplying that event info (example from TheHeader.vue):
```
v-kv-track-event="'TopNav|click-Portfolio-My public Trustee page'"
```